docker-tests/docker-tests.sh

```diff
@@ -0,0 +1,203 @@
+#! /usr/bin/env bash
```

Comment:
Use this one
https://raw.githubusercontent.com/open-io/ansible-role-openio-skeleton/docker-tests/docker-tests.sh